llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -100,6 +100,8 @@ static cl::opt<unsigned> MemoryCheckMergeThreshold(
 
 /// Maximum SIMD width.
 const unsigned VectorizerParams::MaxVectorWidth = 64;
+/// Maximum LMUL factor.
```

Comment:
This is just the unroll factor, except when using EVL vectorization?

Reply:
Yes, fixed the comment